games-action/polymc/polymc-1.4.2.ebuild

```diff
@@ -0,0 +1,129 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake java-pkg-2 optfeature xdg
+
+HOMEPAGE="https://polymc.org/ https://github.com/PolyMC/PolyMC"
+DESCRIPTION="A custom, open source Minecraft launcher"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="
+		https://github.com/PolyMC/PolyMC
+		https://github.com/PolyMC/libnbtplusplus
+	"
+
+	EGIT_SUBMODULES=( 'depends/libnbtplusplus' )
+else
+	MY_PN="PolyMC"
+
+	# Let's use the vendored tarball to avoid dealing with the submodules directly
+	SRC_URI="
+		https://github.com/PolyMC/PolyMC/releases/download/${PV}/${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz
+	"
+
+	# The PolyMC's files are unpacked to ${WORKDIR}/PolyMC-${PV}
+	S="${WORKDIR}/${MY_PN}-${PV}"
+
+	KEYWORDS="~amd64"
+fi
+
+# Apache-2.0 for MultiMC (PolyMC is forked from it)
+# GPL-3 for PolyMC
+# LGPL-3 for libnbtplusplus
+# See the rest of PolyMC's libraries at https://github.com/PolyMC/PolyMC/tree/develop/libraries
+LICENSE="Apache-2.0 Boost-1.0 BSD BSD-2 GPL-2+ GPL-3 LGPL-3 OFL-1.1 MIT"
```

Comment:
optional-bare got dropped, you can remove the boost-1.0 hers

Reply:
yes, i forgot about this field :sweat_smile: